apps/shell/test/allowlist.test.js

```diff
@@ -1,14 +1,66 @@
 /**
- * Allowlist helper function tests
+ * Helper functions tested
  */
 const helpers = require('../utils/helpers')
 
-/**
- * Constants used during test
- */
-const ALLOW_LIST = new Set(['owens.osc.edu', 'pitzer.osc.edu', '*.ten.osc.edu'])
+//generate_host_allowlist
+describe('Helper function generate_host_allowlsit', () => {
```

Comment:
```suggestion
describe('Helper function generate_host_allowlist()', () => {
```
Typo, also add `()` to the test name to indicate which function is being tested